src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/StartGeneralApplicationBusinessProcessTaskHandler.java

```diff
@@ -51,31 +56,40 @@ public VariableMap getVariableMap() {
     }
 
     private CaseData startGeneralApplicationBusinessProcess(ExternalTask externalTask) {
-        ExternalTaskInput externalTaskInput = mapper.convertValue(externalTask.getAllVariables(),
+        ExternalTaskInput externalTaskInput = null;
+        try {
+            externalTaskInput = mapper.convertValue(externalTask.getAllVariables(),
                                                                   ExternalTaskInput.class);
-        String caseId = externalTaskInput.getCaseId();
-        CaseEvent caseEvent = externalTaskInput.getCaseEvent();
-        StartEventResponse startEventResponse = coreCaseDataService.startUpdate(caseId, caseEvent);
-        CaseData data = caseDetailsConverter.toCaseData(startEventResponse.getCaseDetails());
-        List<Element<GeneralApplication>> generalApplications = data.getGeneralApplications();
+        } catch (ValueMapperException | IllegalArgumentException e) {
+            throw new InvalidCaseDataException("Mapper conversion failed due to incompatible types", e);
+        }
+        if (null != externalTaskInput) {
```

Comment:
This `if` shouldn't be necessary. As the exception gets catched and re-thrown in theory that value can never be `null`.

If the compiler or SonarLint are complaining, try using `Objects.requireNonNull`.

```
        try {
            externalTaskInput = Objects.requireNonNull(mapper.convertValue(externalTask.getAllVariables(),
                                                                  ExternalTaskInput.class));
        } catch (ValueMapperException | IllegalArgumentException | NullPointerException e) {
            throw new InvalidCaseDataException("Mapper conversion failed due to incompatible types", e);
        }
```
Even better if you put the `NullPointerException` in its own `catch` clause as it's not really a matter of incompatible types but of data not being provided altogether.